src/js/qg-main/qg-error-handler.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * This plugin
+ * 	-checks for error messages on page
+ * 	-generates a hidden form with error information
+ * 	-submits to form submission handler which formats and sends error information to support team
+ * */
+
+(function ($, swe) {
+  "use strict";
+  swe.handleErrors = function (selector) {
+    var errorInfoContainer = $(selector),
+      errorInfo = [],
+      generateForm = function (errorInfo) {
+        var form = $("<form/>", {
+          method: "POST",
+          action:
+            "https://www.smartservice.qld.gov.au/services/submissions/email/qld/qld-error-notifications",
```

Comment:
Can we test that this form will 'works'

Reply:
it could, but that mean required adding testing facility to the repo
and the person added the code will reponsible for writing test case as well
probably something to ask @manjeetkgov if they want to do so or not

Reply:
was meaning, do we have this in our service catalog to be checked, once and a while.